src/applets/budgie-menu/BudgieMenuWindow.vala

```diff
@@ -35,7 +35,7 @@ public class BudgieMenuWindow : Budgie.Popover {
 	protected CategoryButton all_categories;
 
 	/* Mapped id -> button */
-	protected HashTable<string,MenuButton?> menu_buttons = null;
+	protected HashTable<string, MenuButton?> menu_buttons = null;
```

Comment:
The lack of whitespace for key/val map (HashTable) type declaration is intentional.